ci/www/public/_redirects

```diff
@@ -1,9 +1,13 @@
+# Redirect materialize.io to materialize.com
```

Comment:
Calling out this file as the one thing that's not a straightforward search-and-replace

Reply:
Reference: https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file